ubidots/ubidots_in.html

```diff
@@ -50,78 +50,78 @@
       </div>
       <!-- VARIABLE 1 -->
       <div id="variable-1" class="form-row" style="margin-bottom:10px">
```

Comment:
Why not changing this to a flexbox container so we can center everything vertically without any hassle?